code/script.js

```diff
@@ -1,44 +1,154 @@
       <section class="bot-msg">
         <img src="assets/bot.png" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
       </section>
-    `
+    `;
   }
   // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
   chat.scrollTop = chat.scrollHeight;
-}
+};
 
+//const nextQuestion = (message) => {};
 // Starts here
 const greetUser = () => {
   // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot');
+  showMessage("Hello there, What's your name?", "bot");
   // Just to check it out, change 'bot' to 'user' here 👆
-}
+};
+
+// Store the value in a variable so I can access it after we
+
+// Initial button click, here I should get the name entered
+sendButton.addEventListener("click", (event) => {
+  event.preventDefault();
+  userName = name.value; //username is here reassigned to mean what we want it to mean in this specific function
+  showMessage(`${userName}`, "user");
+  name.value = ""; // clear it from the input
+  setTimeout(() => askZodiacSigns(), 1000); //calls the function askZodiac
+});
+
+//Here I call the function where I welcome the user and ask them to tell me their zodiac sign
+const askZodiacSigns = () => {
+  console.log("this works");
+  showMessage(
+    `OMG ${userName} is my favourite name in the whole world 🥳!! What's your zodiac sign?`,
+    "bot"
+  );
+  setTimeout(() => nextQuestion(), 1000); //calls the function nextQuestion
+};
+
+const nextQuestion = () => {
+  //this function is whats defining what we meen with the function nextQuestion
+  console.log("next question works!");
+  /*  showMessage(´next question is working´ , "user"); */
+
+  inputWrapper.innerHTML = `
+  <form id="zodiac-form">
+          <label for="zodiac-form">Zodiac</label> 
+          <input id="zodiac-input" type="text" />
+          <button class="send-btn" id="send-zodiacform" type="submit">Send</button>
+  </form>
+   `;
+  botZodiacReply();
+};
+
+//This is all one and the same funciton (botZodiacReply)
+
+const botZodiacReply = () => {
+  console.log("bot reply works");
+  const zodiacForm = document.getElementById("zodiac-form"); //declaring the variables for the form
+  const zodiacInput = document.getElementById("zodiac-input"); //declaring the variables for the input of the form
+  const sendZodiacForm = document.getElementById("send-zodiacform"); //declaring the variables for the input of the send zodiac form (button)
+
+  sendZodiacForm.addEventListener("click", (event) => {
+    event.preventDefault();
+    console.log("zodiac buttton is working");
+    const zodiacSign = zodiacInput.value;
+    showMessage(
+      `Oh fantasctic! ${zodiacSign} are amazing at doing affirmation work 🌟 So you are basically half way there 😉!`,
+      "bot"
+    );
```

Comment:
Could this be tidied up so "bot" isn’t on a line by itself? Makes it easier to read.